views/decode.hbs

```diff
@@ -11,15 +11,15 @@
         <div class="container">
             <header class="d-flex flex-wrap align-items-center justify-content-center justify-content-md-between py-3 mb-4 border-bottom">
                 <a href="/" class="d-flex align-items-center col-md-3 mb-2 mb-md-0 text-dark text-decoration-none">
-                    <span class="fs-4 btn btn-outline-primary me-2">Back to Discovery</span>
+                    <span class="btn btn-outline-secondary me-2">Back</span>
                 </a>
 
-                <ul class="fs-4 nav col-12 col-md-auto mb-2 justify-content-center mb-md-0">
-                    <li><a href="/api/protocol/logger" class="nav-link">Protocol</a></li>
-                </ul>
+                <a href="/api/protocol/logger" class="nav col-12 col-md-auto mb-2 mb-md-0 text-decoration-none">
+                    <span class="btn btn-outline-secondary me-2">Protocol</span>
+                </a>
 
                 <a class="col-md-3 text-end">
-                    <span class="fs-4 btn btn-primary me-2">To Authentication</span>
+                    <span class="btn btn-outline-secondary me-2">Test Authentication</span>
                 </a>
             </header>
```

Comment:
In some views we use nav-elements and in some we use header-elements. To improve maintainance in the future it would be better if we just use one structure.

Reply:
Done